source/PlayState.hx

```diff
@@ -4574,11 +4594,11 @@ class PlayState extends MusicBeatState
 							unlock = true;
 						}
 				}
+			}
 
-				if(unlock) {
-					Achievements.unlockAchievement(achievementName);
-					return achievementName;
-				}
```

Comment:
-221

Reply:
-- literally immediately below it
![image](https://user-images.githubusercontent.com/96351114/166000102-3fc025a2-0809-4b90-b0d3-410aace23189.png)